### PR TITLE
Adds bcrowe/cakephp-api-pagination

### PR DESCRIPTION
**Pull request by bravo-kernel**
Adds https://github.com/bcrowe/cakephp-api-pagination

Refs https://github.com/bcrowe/cakephp-api-pagination/issues/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +274,7 @@ Additional lists you might find useful:
 ## REST and API
 *Plugins and web tools for developing REST-ful APIs.*
 
+- [API Pagination Plugin](https://github.com/bcrowe/cakephp-api-pagination) - Plugin that injects pagination information into serialized JsonView and XmlView responses.
```
Comment (dereuromark) [inline README.md]: The plugin name looks wrong. And you can directly start with `Injects ...` as its clear then it is a plugin.

Comment (bravo-kernel) [inline README.md]: Suggestions for the plugin name? E.g. just "Pagination plugin" ?

Comment (dereuromark) [inline README.md]: The name it has when you load it to avoid confusing :)

**Comment by bravo-kernel** [comment]
Renamed and rebased

**Comment by dereuromark** [comment]
Isn't the name technically `BryanCrowe/...`?

**Comment by bravo-kernel** [comment]
It is but there currently is some lack of naming consistency in the list if you ask me. Some use namespace, others don't. I just followed the other entries in the `REST and Api` section.

**Pull request merged**